engine/standalone/engine/src/test/scala/pl/touk/nussknacker/engine/standalone/StandaloneProcessConfigCreator.scala

```diff
@@ -242,12 +242,13 @@ class StandaloneFilterWithLog(filterExpression: LazyParameter[java.lang.Boolean]
       implicit val implicitLpi: LazyParameterInterpreter = lpi
       val lazyLogInformation = filterExpression.map(StandaloneLogInformation(_))
       val exprInterpreter: (ExecutionContext, engine.api.Context) => Future[StandaloneLogInformation] = lpi.createInterpreter(lazyLogInformation)
-      (ctx: engine.api.Context, ec: ExecutionContext) => {
+      (ctxs: List[engine.api.Context], ec: ExecutionContext) => {
         implicit val ecc: ExecutionContext = ec
-        exprInterpreter(ec, ctx).flatMap(exprResult => {
-          if(exprResult.filterExpression) continuation(ctx, ec)
-          else Future(Right(List(InterpretationResult(DeadEndReference(nodeId.id), exprResult, ctx))))
-        })
+        Future.sequence(ctxs.map(ctx => exprInterpreter(ec, ctx).map((_, ctx)))).flatMap { runInformations =>
+          val (pass, skip) = runInformations.partition(_._1.filterExpression)
+          val skipped = skip.map(info => EndResult(InterpretationResult(DeadEndReference(nodeId.id), info._1, info._2)))
```

Comment:
can you use:
```
map {
  case (xxx, yyy) => 
}
``` syntax instead of _1 and _2 so it will be more clear what happens here without using IDE?

Reply:
:+1: